pyramid_learning_journal/production.ini

```diff
@@ -0,0 +1,63 @@
+###
+# app configuration
+# http://docs.pylonsproject.org/projects/pyramid/en/latest/narr/environment.html
+###
+
+[app:main]
+use = egg:pyramid_learning_journal
+
+pyramid.reload_templates = false
+pyramid.debug_authorization = false
+pyramid.debug_notfound = false
+pyramid.debug_routematch = false
+pyramid.default_locale_name = en
+
+sqlalchemy.url = DATABASE_URL
```

Comment:
You're overriding this setting anyway in your `__init__.py` file, but this is not a proper way to set the `sqlalchemy.url` configuration variable. Your `production.ini` has no idea that your `DATABASE_URL` is referring to an environment variable. It just thinks you're trying to set the string of `DATABASE_URL` as the value for the `sqlachemy.url`